lib/config/error-codes/api.json

```diff
@@ -1,5 +1,5 @@
 {
-  "code": 1,
+  "code": 2,
```

Comment:
... why do you change the domain codes of API and external errors?

Reply:
I just re orders the domains